source/_patterns/02-molecules/leads/lead-gallery.twig

```diff
@@ -1,23 +1,33 @@
+{% set lead_capton %}
```

Comment:
looks like a small typo here. should be `caption`?

Reply:
Yup, updating.